README.rst

```diff
@@ -193,39 +193,43 @@ Features
     input code      full name
 
       ar              Arabic
-      ru              Russian
-      nl              Dutch
+      be              Belarusian
+      bg              Bulgarian
+      da              Danish
       de              German
+      el              Greek
       en              English
       es              Spanish
+      et              Estonian
+      fa              Persian
+      fi              Finnish
       fr              French
       he              Hebrew
+      hi              Hindi
+      hr              Croatian
+      hu              Hungarian
+      id              Indonesian
       it              Italian
+      ja              Japanese
       ko              Korean
+      lt              Lithuanian
+      mk              Macedonian
```

Comment:
Please delete this extra line, since your PR only adds Thai as a new language, I think you accidentally included a few other languages